contracts/Vault.sol

```diff
@@ -228,7 +273,7 @@ contract Vault is
     function claimYield(address _to) external override(IVault) nonReentrant {
         require(_to != address(0), "Vault: destination address is 0x");
 
-        uint256 yield = yieldFor(_msgSender());
+        uint256 yield = _yieldForWithPerfFee(_msgSender());
```

Comment:
What if we just changed the `yieldFor` function to return `(uint256 yield, uint256 perfFee)` instead? I

Reply:
updated